programs/gateway/src/instructions/create_voter_weight_record.rs

```diff
@@ -0,0 +1,63 @@
+use crate::state::*;
+use anchor_lang::prelude::*;
+use anchor_spl::token::Mint;
+use spl_governance::state::realm;
+
+/// Creates VoterWeightRecord used by spl-gov
+/// This instruction should only be executed once per realm/governing_token_mint/governing_token_owner
+/// to create the account
+#[derive(Accounts)]
+#[instruction(governing_token_owner: Pubkey)]
+pub struct CreateVoterWeightRecord<'info> {
+    #[account(
+        init,
+        seeds = [ b"voter-weight-record".as_ref(),
+                realm.key().as_ref(),
+                realm_governing_token_mint.key().as_ref(),
+                governing_token_owner.as_ref()],
```

Comment:
More of an aside, but is there a standard for how to generate voter weight record addresses?  That would be really useful from a UI perspective, I imagine